contentpacks/khanacademy.py

```diff
@@ -567,14 +568,56 @@ def retrieve_kalite_data(lang="en", force=False) -> list:
 
     url = url.format(projection=json.dumps(projection), lang=lang)
 
-    node_data_path = download_and_clean_kalite_data(url, lang=lang, ignorecache=force, filename="nodes.json")
+    node_data_path = download_and_clean_kalite_data(url, lang=lang, ignorecache=force, filename=filename)
 
     with open(node_data_path, 'r') as f:
         node_data = ujson.load(f)
+    
+    if not lang == "en" and not no_dubbed_videos == True:
+        node_data = addin_dubbed_video_mappings(node_data, lang) 
 
     return node_data
 
 
+def addin_dubbed_video_mappings(node_data, lang="en"):
+    # Get the dubbed videos from the spreadsheet and substitute them 
+    # for the video, and topic attributes of the returned data struct.
+
+    # Get the list of video ids from dubbed video mappings
+    lang_name = get_lang_name(lang).lower()
+    dubbed_videos_data = pkgutil.get_data('contentpacks', "resources/dubbed_video_mappings.json")
```

Comment:
`pkgutil.get_data` is only meant for retrieving files bundled into packages, i.e. things checked into source. Since `dubbed_vdeo_mappings.json` is created at runtime, we should instead just write it to the `build` folder and read it from there.